.github/workflows/build-and-test.yaml

```diff
@@ -21,15 +21,21 @@ on:
   push:
     branches:
       - main
-    paths:
-      - '!**.md'
 
 jobs:
   build:
     runs-on: ubuntu-18.04
     timeout-minutes: 90
     steps:
-      - uses: actions/checkout@v2
+      - name: Set up Go 1.14
```

Comment:
Let's at least set go 1.14 and 1.15

Reply:
Just FYI @EvanLjp , make a build matrix like [this](https://github.com/apache/skywalking-python/blob/606a005fa046583d13922ec27f9b2b0fa58fe860/.github/workflows/build.yaml#L29-L41)

Reply:
@kezhenxu94 could u help me?  I may not be familiar with the Github Workflow structure and wonder how to validate both go versions at the same time.